pages/support.tsx

```diff
@@ -0,0 +1,301 @@
+`;
+
+export default function Support() {
+  return (
+    <>
+      <Head>
+        <title>Support | Holaplex</title>
+        <meta
+          property="description"
+          key="description"
+          content="Our mission is to empower creators and collectors with a suite of tools to create, market, and sell NFTs."
+        />
+        <body />
+      </Head>
+
+      <HeroWrapper>
+        <Section>
+          <Content className="text-center lg:my-8">
+            <div style={{ maxWidth: '1044px' }} className='mx-auto'>
+              <Heading>Solanas top community <br className='hidden lg:block' /> supported NFT platform</Heading>
+              <Paragraph>
+                Web3 is an exciting place but sometimes we need help. We rely on collaboration from members from the community to provide assistance. Get involved by contributing to our wiki page or in our discord.
+              </Paragraph>
+              <a href='https://docs.holaplex.com/'><Button>Read The Docs</Button></a>
+            </div>
+          </Content>
+        </Section>
+      </HeroWrapper>
+
+      <div className="container mx-auto px-6 md:px-12">
+        <Section>
+          <div className="w-full lg:w-1/2">
+            <Content>
+              <SubHeading>Need Help?</SubHeading>
+              <Heading>Get support from the community</Heading>
+            </Content>
+          </div>
+          <Content>
+            <div className='lg:flex lg:gap-4 w-full'>
+              <div>
+                <Paragraph>
+                  <b>Information at your fingertips</b><br />
+                  Our documentation is expanding by the day thanks to contributions from our community.<br />
+                  <a href="https://wiki.holaplex.com/" style={{ color: '#F7CC34' }}>Find your answer</a>
+                </Paragraph>
+              </div>
+              <div>
+                <Paragraph>
+                  <b>Get help from the community</b><br />
+                  Couldn’t find your answer from the documentation? Ask a member from the community.<br />
+                  <a href="https://discord.com/invite/holaplex" style={{ color: '#F7CC34' }}>Join the Discord</a>
+                </Paragraph>
+              </div>
+            </div>
+          </Content>
+        </Section>
+
+        <Section>
+          <Content className="text-center">
+            <SubHeading>What product do you need help with?</SubHeading>
+            <Heading>Support for all of the <br className='hidden lg:block' /> Holaplex NFT products</Heading>
+          </Content>
+
+          <Content>
+            <SupportRow>
+              <SupportImage>
+                <SupportBlob style={{ backgroundImage: 'url(/images/page-support/blob-02.svg)' }} />
+                <div className='rounded-lg lg:rounded-2xl w-full overflow-hidden border border-gray-500 z-10 relative aspect-video'>
+                  <Image alt='' src={'/images/page-support/mint.jpg'} layout='fill' />
```

Comment:
provide an `alt`

Reply:
does `src` need to be in `{...}`?

Reply:
It does not unless you import an image from the filesystem, I've gone ahead and changed it to just quotes.